src/subscription_manager/exceptions.py

```diff
@@ -149,6 +156,18 @@ def format_rate_limit_exception(self, rate_limit_exception, _):
         else:
             return RATE_LIMIT_MESSAGE
 
+    def format_cert_loading_error(self, exc: CertificateLoadingError, _: str):
+        message = exc.args[0]
+        file = exc.args[1]
+        pem = exc.args[2]
+        if file is not None:
+            if file.startswith(DEFAULT_PRODUCT_CERT_DIR):
+                return PRODUCT_CERTIFICATE_LOADING_PATH_ERROR.format(file=file, message=message)
+            return CERTIFICATE_LOADING_PATH_ERROR.format(file=file, message=message)
+        if pem is not None:
+            return CERTIFICATE_LOADING_PEM_ERROR.format(data=terminal_printable_content(pem), message=message)
+        return CERTIFICATE_LOADING_ERROR.format(file=file)
```

Comment:
This is wrong `CERTIFICATE_LOADING_ERROR` does not contain any `{file}`. Maybe it is not the best idea to have templates outside this method.